ReactSkia/components/RSkComponentText.cpp

```diff
@@ -55,7 +56,20 @@ void RSkComponentParagraph::OnPaint(SkCanvas *canvas) {
       if(!parent->expectedAttachmentCount || (parent->expectedAttachmentCount == parent->currentAttachmentCount)) {
             auto frame = parent->getAbsoluteFrame();
             paragraph->layout(frame.size.width);
-            paragraph->paint(canvas, frame.origin.x, frame.origin.y);
+            SkScalar textHeight = paragraph->getHeight();
+            uint32_t attachmentCount = 0;
+
+            for(auto &fragment: data.attributedString.getFragments()) {
```

Comment:
since its the same logic for nested and non-nested texts ; can this piece of code moved to separate/inline function ?

Reply:
yes, added changes.